File: sdkwork-app-sdk-typescript/src/types/qr-code-status-vo.ts
import type { LoginVO } from './login-vo';
import type { UserInfoVO } from './user-info-vo';

/** 二维码状态响应 */
export interface QrCodeStatusVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 二维码状态: pending/scanned/confirmed/expired */
  status?: 'pending' | 'scanned' | 'confirmed' | 'expired';
  /** 用户信息(扫描后返回) */
  userInfo?: UserInfoVO;
  /** 登录令牌(确认后返回) */
  token?: LoginVO;
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/QrCodeStatusVO.java
package com.sdkwork.app.model;

public class QrCodeStatusVO {
    private String createdAt;
    private String updatedAt;
    private String status;
    private UserInfoVO userInfo;
    private LoginVO token;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public String getStatus() {
        return this.status;
    }
    
    public void setStatus(String status) {
        this.status = status;
    }

    public UserInfoVO getUserInfo() {
        return this.userInfo;
    }
    
    public void setUserInfo(UserInfoVO userInfo) {
        this.userInfo = userInfo;
    }

    public LoginVO getToken() {
        return this.token;
    }
    
    public void setToken(LoginVO token) {
        this.token = token;
    }
}
